buildSrc/src/main/java/org/elasticsearch/gradle/test/RestApiSpecExtension.java

```diff
@@ -0,0 +1,70 @@
+package org.elasticsearch.gradle.test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RestApiSpecExtension {
+    private List<String> includesCoreSpec = new ArrayList<>();
+    private List<String> includesCoreTests = new ArrayList<>();
+    private List<String> includesXpackSpec = new ArrayList<>();
+    private List<String> includesXpackTests = new ArrayList<>();
+    private boolean alwaysCopySpec = false;
+    private boolean copyTests = false;
+    private boolean copyXpackTests = false;
+
+    public void includeCoreSpec(String include) {
+        includesCoreSpec.add(include);
+    }
+
+    public List<String> getIncludesCoreSpec() {
+        return includesCoreSpec;
+    }
+
+    public void includeCoreTests(String include) {
+        includesCoreTests.add(include);
+    }
+
+    public List<String> getIncludesCoreTests() {
+        return includesCoreTests;
+    }
+
+    public void includeXpackSpec(String include) {
+        includesXpackSpec.add(include);
+    }
+
+    public List<String> getIncludesXpackSpec() {
+        return includesXpackSpec;
+    }
+
+    public void copyCoreTests(boolean copyTests) {
```

Comment:
We should use a typical `setter/getter` pattern here. I think configuring  like`copyCoreTests = true` is more idiomatic.

Reply:
I ended up removing this from the PR in favor of just using an include pattern.